script/coverage

```diff
@@ -38,11 +38,13 @@ grep -r -I --line-number "# pragma: +no.*cover" $SOURCE_DIR && {
   exit 1
 }
 
-coverage run --branch --source=$SOURCE_DIR "$(command -v nosetests)" --with-no-network --with-xunit "$@"
-coverage html
-coverage xml
-coverage report --show-missing
-coverage report | grep ^TOTAL | grep -qv 100% && {
-    echo "Incomplete code coverage" >&2
-    exit 1
-} || echo "Code coverage 100%"
+pytest \
+        --disable-network \
+        --cov-reset \
+        --cov=$SOURCE_DIR \
+        --cov-fail-under=100 \
```

Comment:
Nice. I'd only ported over to pytest, e.g. https://github.com/octodns/octodns-azure/pull/3, and hadn't really looked into what it could do differently/better. Nice this stuff is integrated. I'll look at updating the octodns-template bits to match this and then new stuff will use it and I have some WIP tooling to make changes across all the repos easy-ish.